test/apps/test_fuzzy_match.py

```diff
@@ -1,36 +1,51 @@
 #!/usr/bin/python2
 
+#
+# Copyright (C) 2021 NVIDIA CORPORATION & AFFILIATES. ALL RIGHTS RESERVED.
+#
+# See file LICENSE for terms.
+
 import os
 import commands
 import re
 import argparse
 import sys
 import logging
 
-###################
-# Main test runner
-###################
-class FuzzyTester:
-    def __init__(self, ucx_info, verbose):
-        self.ucx_info = ucx_info
-        if verbose:
-            logging.basicConfig(level=logging.DEBUG)
+class FuzzyTest(object):
```

Comment:
I think this class has nothing to do with "test" or "expectation".
It simply modifies the env vars, according to params from user.
Maybe rename it (including members, vars, methods) accordingly?

Reply:
it also handles expected test results, which is not related to env vars